src/test/java/ay2122s1_cs2103t_w16_2/btbb/testutil/OrderBuilder.java

```diff
@@ -8,14 +11,23 @@
  */
 public class OrderBuilder {
     private static final String DEFAULT_PHONE = "85355255";
+    private static final String DEFAULT_NAME = "Amy Bee";
+    private static final String DEFAULT_ADDRESS = "123, Jurong West Ave 6, #08-111";
+    private static final int DEFAULT_INDEX = 1;
 
+    private Index clientIndex;
```

Comment:
Order builder does not need clientIndex. The Order model does not have clientIndex.